rqt_gui/src/rqt_gui/main.py

```diff
@@ -53,10 +53,18 @@ def main(self, argv=None, standalone=None, plugin_argument_provider=None):
 
         # ignore ROS specific remapping arguments
         argv = rclpy.utilities.remove_ros_args(args=argv)
-
-        return super(
-            Main, self).main(argv, standalone=standalone,
-                             plugin_argument_provider=plugin_argument_provider)
+        help_text = """<p><b>rqt</b> is a GUI framework that is able to load various plug-in tools
+as dockable windows. There are currently no plug-ins selected. To add plug-ins, select items from
+the <b>Plugins</b> menu.</p>
+<p>You may also save a particular arrangement of plug-ins as a <i>perspective</i> using the
+<b>Perspectives</b> menu.
+<p>See <a href="http://wiki.ros.org/rqt/">the rqt Wiki page</a> for more information</p>
```

Comment:
So I'd really like not to point to the ROS 1 wiki anymore.  However, we don't have great documentation about RQt on https://docs.ros.org .  My suggestion here is to just remove this sentence, and then open a new issue on https://github.com/ros2/ros2_documentation to port RQt documentation over there.  Once that is in, we can always revisit this and add the link back.

Reply:
Removed the link to the ROS 1 Wiki and added an issue to the ROS 2 documentation: https://github.com/ros2/ros2_documentation/issues/2415